### DISCUSSION TITLE
Downloading

### DISCUSSION THREAD

**Discussion by xyberxvii** [minbrowser/min#2251] [Ideas]
a "Issue" i have with min browser is whenever im downloading something and i forget it and i close the app the download will be abroaded. However almost every other browser does give a warning to ask if you really want to close the browser. I would love to see this implementet in Min

**Comment by mikeyww** [comment]
Also:

1. Would add a way to **view the downloads in progress** and the **download history**.
2. Would use a **temporary file extension** (e.g., ".mdownload") while a download is in progress.
